core/templates/dev/head/pages/donate/Donate.js

```diff
@@ -18,10 +18,11 @@
 
 oppia.controller('Donate', [
   '$http', '$scope', '$timeout', '$window', 'UrlInterpolationService',
-  'siteAnalyticsService',
+  'siteAnalyticsService', 'windowDimensionsService',
   function(
       $http, $scope, $timeout, $window, UrlInterpolationService,
-      siteAnalyticsService) {
+      siteAnalyticsService, windowDimensionsService) {
+    $scope.windowIsNarrow = (windowDimensionsService.getWidth() <= 800);
```

Comment:
Why not use the function in ``windoweDimensionsService`` to determine this, instead?